docs/getting-started-guides/minikube.md

```diff
@@ -279,6 +279,12 @@ $ minikube start --docker-env HTTP_PROXY=http://$YOURPROXY:PORT \
                  --docker-env HTTPS_PROXY=https://$YOURPROXY:PORT
 ```
 
+Your Virtual Machine address might be 192.168.99.100, chances are your proxy settings will prevent kubectl to directly reach it.
+To by-pass proxy configuration for this IP address you should modify your no_proxy settings, you can do so with:
```

Comment:
suggested edit: "To by-pass proxy configuration for this IP address, you should modify your no_proxy settings. You can do so with:"